disco_aws_automation/disco_ssm.py

```diff
@@ -0,0 +1,175 @@
+"""
+Manage AWS SSM document creation and execution
+"""
+import os
+import logging
+import time
+import json
+
+import boto3
+
+from botocore.exceptions import ClientError
+from . import read_config
+from .resource_helper import throttled_call, wait_for_state_boto3
+from .exceptions import TimeoutError
+
+logger = logging.getLogger(__name__)
+
+
+SSM_DOCUMENTS_DIR = "ssm/documents"
+SSM_EXT = ".ssm"
+SSM_WAIT_TIMEOUT = 5 * 60
+SSM_WAIT_SLEEP_INTERVAL = 15
+DOCUMENT_PREFIX = "asiaq-"
```

Comment:
Talked to Johnathan on the side, seems like the only good reason for specifying the document prefixes in asiaq is just to filter out the AWS documents. In the case, we'll just hardcode the prefix for AWS documents in asiaq, which is `AWS-`, and filter them out that way instead.